hack/cluster-sync.sh

```diff
@@ -6,9 +6,14 @@ ${KUBECTL} delete --ignore-not-found -f ${local_handler_manifest}
 # Set debug verbosity level for logs when using cluster-sync
 sed "s#--v=production#--v=debug#" ${local_handler_manifest} | ${KUBECTL} create -f -
 
-desiredNumberScheduled="$(${KUBECTL} get daemonset -n nmstate nmstate-handler -o=jsonpath='{.status.desiredNumberScheduled}')"
 for i in {300..0}; do
-	if [ $desiredNumberScheduled == "$(${KUBECTL} get daemonset -n nmstate nmstate-handler -o=jsonpath='{.status.numberAvailable}')" ]; then
+
```

Comment:
please drop the line, it is already indented, no need to divide the block even more